src/compile/scale/domain.ts

```diff
@@ -191,9 +192,9 @@ function normalizeDomain(domain: VgDomain): (any[] | VgDataRef)[] {
         data: d.data
       };
     });
+  } else {
+    throw 'Should not get here.';
```

Comment:
But that would make one branch untested (hidden else for that case). 

(Yep, in the past I was dumb and didn't do this, but will follow this pattern going forward.)